Configuration/FlexForms/YellowPages.xml

```diff
@@ -53,25 +41,21 @@
 									<parentField>parent</parentField>
 								</treeConfig>
 							</config>
-						</TCEforms>
 					</settings.presetTrade>
 					<settings.district>
-						<TCEforms>
-							<label>LLL:EXT:yellowpages2/Resources/Private/Language/FlexForms.xlf:district</label>
-							<config>
+						<label>LLL:EXT:yellowpages2/Resources/Private/Language/FlexForms.xlf:district</label><config>
 								<type>select</type>
 								<renderType>selectSingle</renderType>
 								<foreign_table>tx_yellowpages2_domain_model_district</foreign_table>
 								<items type="array">
 									<numIndex index="0">
-										<numIndex index="0">LLL:EXT:yellowpages2/Resources/Private/Language/locallang_db.xlf:tx_yellowpages2_domain_model_company.district.pleaseChoose</numIndex>
-										<numIndex index="1" />
+										<label>LLL:EXT:yellowpages2/Resources/Private/Language/locallang_db.xlf:tx_yellowpages2_domain_model_company.district.pleaseChoose</label>
```

Comment:
With this change the extension is not TYPO3 11 compatible anymore

Reply:
Done